packages/projects-docs/pages/learn/_meta.json

```diff
@@ -1,11 +1,27 @@
 {
-  "introduction": "Introduction",
-  "getting-started": "Getting Started",
-  "sandboxes": "Sandboxes",
-  "repositories": "Repositories",
-  "boxy": "Boxy",
-  "environment": "Environment",
-  "integrations": "Integrations",
-  "teams": "Team and access",
-  "plan-billing": "Plans and Billing"
+  "index": "Introduction",
+  "getting-started": "Guides",
+  "-- cloud-development": {
+    "type": "separator",
+    "title": "Cloud Development"
+  },
+  "repositories": "",
+  "sandboxes": "",
+  "editors": "",
+  "boxy": "",
+  "environment": "",
+  "integrations": "",
+  "-- browser-sandboxes": {
+    "type": "separator",
+    "title": "Browser Sandboxes"
+  },
+  "browser-vs-cloud-sbx": "Why Browser Sandboxes",
+  "browser-sandboxes": "Features",
```

Comment:
This sounds vague. We should add something more descriptive.

Reply:
good point, I'll fix this